Zilon.Core/Zilon.Core/Tactics/Behaviour/AttackTask.cs

```diff
@@ -20,7 +20,8 @@ protected override void ExecuteTask()
         {
             if (!Target.CanBeDamaged())
             {
-                throw new InvalidOperationException("Попытка атаковать цель, которой нельзя нанести урон.");
+                return;
+                //throw new InvalidOperationException("Попытка атаковать цель, которой нельзя нанести урон.");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5544638)